private-templates-service/client/src/components/Dashboard/SearchPage/Filter/Filter.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
+import { RootState } from "../../../../store/rootReducer";
+import { connect } from 'react-redux';
+import { StyledFilterDropdown } from './styled';
+import { clearFilter, queryFilter } from "../../../../store/filter/actions";
+import { THEME } from '../../../../globalStyles';
+
+const mapStateToProps = (state: RootState) => {
+  return {
+    isSearch: state.search.isSearch,
+    filterType: state.filter.filterType
+  }
+}
+
+const mapDispatchToProps = (dispatch: any) => {
+  return { 
+    clearFilter: () => {
+      dispatch(clearFilter());
+    },  
+    queryFilter: (filterType: string) => {
+      dispatch(queryFilter(filterType));
+    }
+  }
+}
+
+interface Props {
+  isSearch: boolean;
+  filterType: string; 
+  queryFilter: (filterType: string) => void;
+  clearFilter: () => void;
+}
+
+const options: IDropdownOption[] = [
+  {key: 'created by me', text: "Created by me"},
```

Comment:
Have we implemented some sort of localization yet?

Reply:
We have not implemented any localization yet, we will be looking into it soon.